src/Tickets/Seating/Commerce/Controller.php

```diff
@@ -56,8 +57,59 @@ public function unregister(): void {
 			[ $this, 'filter_timer_token_object_id_entries' ],
 		);
 		remove_filter( 'tribe_tickets_ticket_inventory', [ $this, 'get_seated_ticket_inventory' ] );
+		remove_filter( 'tec_tickets_get_ticket_counts', [ $this, 'set_event_stock_counts' ] );
 		remove_action( 'updated_postmeta', [ $this, 'sync_seated_tickets_stock' ] );
 	}
+	
+	/**
+	 * Sets the stock counts for the event.
+	 *
+	 * @since TBD
+	 *
+	 * @param array<string,array<string|int>> $types  The types of tickets.
+	 * @param int                             $post_id The post ID.
+	 *
+	 * @return array<string,array<string|int>> The types of tickets.
+	 */
+	public function set_event_stock_counts( $types, $post_id ): array {
+		$has_seating = get_post_meta( $post_id, Meta::META_KEY_ENABLED, true );
+		
+		if ( ! $has_seating ) {
+			return $types;
+		}
```

Comment:
Why not use `tec_tickets_seating_enabled`. Important note that mentioned function is checking against `Meta::META_KEY_LAYOUT_ID` instead. If we are incosistent accross the database as to whether a post has seating enabled this can create issues. Rather use one single function for that, and if we need to change it in the future to only have to change that.